_topic_maps/_topic_map.yml

```
@@ -538,6 +538,9 @@ Topics:
 - Name: Customizing the web console
   File: customizing-the-web-console
   Distros: openshift-enterprise,openshift-origin
+- Name: Dynamic plugin
```

Should this be plural?

```suggestion
- Name: Dynamic plugins
```